auth_signup_verify_email/controllers/main.py

```diff
@@ -52,6 +56,12 @@ def passwordless_signup(self, values):
                 _("That does not seem to be an email address."),
             )
             return request.render("auth_signup.signup", qcontext)
+        except EmailUndeliverableError as error:
+            qcontext["error"] = _(str(error))
```

Comment:
You don't need translate the variables
I mean, you can delete the underscore method here `_()`

Reply:
Good catch.
Thanks @moylop260